releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 5.3.0-preview - July 27, 2022
+* Added the following new API for the App Configuration geo-replication support.
```

Comment:
NIT: we can add a reference to issue #178.

Reply:
Do we want to close the issue 178 as well?

Reply:
Not yet. I will wait till GA. It's used to track our roadmap.